docs/tutorials/python/tutorial_scripts/json_schema.py

```diff
@@ -97,10 +90,17 @@
     else:
         raise e
 
+schema.store(schema_body=updated_schema, version=NEW_VERSION)
+schema.get_body(version=VERSION)
 # 4. Bind the JSON schema to the folder
-schema_uri = ORG_NAME + "-" + SCHEMA_NAME + "-" + VERSION
+# Retrieve test project
+PROJECT_ENT = Project(name="My uniquely named project about Alzheimer's Disease").get()
```

Comment:
Could strings like this get put at the top of the script with a disclaimer that the user running through the tutorials should set it to their unique values?

Reply:
Done, that was on me.  I shifted it down